Assets/Scripts/Internal/DaggerfallBillboard.cs

```diff
@@ -82,36 +79,32 @@ void Start()
                     // Example is the treasury in Daggerfall castle, some action records flow through the quest item marker
                     meshRenderer.enabled = false;
                 }
-            }
-        }
 
-        void OnDisable()
```

Comment:
Why was this removed? This pattern is here to restart animations after gameobject is disabled. For example, going inside and outside of a building will disable then enable billboard objects. With this removed in your PR, animating billboards are now broken when going in and out of a building.

Reply:
a8a36c44f5b61423dc50e18460d4dbbeff243db5 reverts this mistake